tools/postgres/schema.sql

```diff
@@ -190,7 +190,8 @@ CREATE TABLE webknossos.organizations(
   _organizationTeam CHAR(24) NOT NULL UNIQUE,
   name VARCHAR(256) NOT NULL,
   created TIMESTAMPTZ NOT NULL DEFAULT NOW(),
-  isDeleted BOOLEAN NOT NULL DEFAULT false
+  isDeleted BOOLEAN NOT NULL DEFAULT false,
+  additionalInformation VARCHAR(2048) NOT NULL DEFAULT ''
```

Comment:
@youri-k could you be convinced to insert the new field before `created` and `isDeleted`? (We have them as the last two fields everywhere). Also in the Scala classes, that would be appreciated.